src/Ledger/Script.lagda

```diff
@@ -159,7 +159,6 @@ record ScriptStructure : Type₁ where
 
   field hashRespectsUnion :
           {A B Hash : Type} → Hashable A Hash → Hashable B Hash → Hashable (A ⊎ B) Hash
-        ⦃ Hash-P1Script ⦄ : Hashable P1Script ScriptHash
```

Comment:
Why is this change necessary?

Reply:
I believe that line was a bug.  It used to be a timelock instance and, for some reason, was changed to a `Hash-P1Script` instance, but that clashes with the other `Hash-P1Script` that was introduced earlier in this module.  If both instances are present in the module, then Agda doesn't know how to choose an instance when the new call to `hash` is made in the following line:
```agda
    ∙  ∀[ s ∈ nonRefScripts ] (hash s ∈ neededHashes → validP1Script witsKeyHashes txvldt s)
```